src/main/java/org/alliancegenome/curation_api/dao/AffectedGenomicModelDAO.java

```diff
@@ -16,13 +17,16 @@ protected AffectedGenomicModelDAO() {
 		super(AffectedGenomicModel.class);
 	}
 	
-	public List<Long> findReferencingDiseaseAnnotations(String agmCurie) {
-		Query jpqlQuery = entityManager.createQuery("SELECT ada.id FROM AGMDiseaseAnnotation ada WHERE ada.subject.curie = :agmCurie");
-		jpqlQuery.setParameter("agmCurie", agmCurie);
-		List<Long> results = (List<Long>) jpqlQuery.getResultList();
+	public List<Long> findReferencingDiseaseAnnotations(Long agmId) {
+		List<Long> results = new ArrayList<>();
 		
-		jpqlQuery = entityManager.createNativeQuery("SELECT diseaseannotation_id FROM diseaseannotation_biologicalentity db WHERE diseasegeneticmodifiers_curie = :agmCurie");
-		jpqlQuery.setParameter("agmCurie", agmCurie);
+		Query jpqlQuery = entityManager.createQuery("SELECT ada.id FROM AGMDiseaseAnnotation ada WHERE ada.subjectBiologicalEntity.id = :agmId");
```

Comment:
Does a findByParam not work here? AGMDiseaseAnnotationDAO.findByParams()
map.put("subjectBiologicalEntity.id", value);

Reply:
Or are you only wanting id's back? We need to create a method with the same interface that returns ids vs full objects.

Reply:
IDs only, but we already have that method (findFilteredIds) - will replace this and similar in other DAOs

Reply:
Ya not a big deal just trying to use the least amount of JQL as possible.